lib/src/providers/VernaProvider/VernaProvider.tsx

```diff
@@ -15,15 +15,22 @@ import type {
   Widget,
   FormProps,
 } from '@rjsf/core';
-import { defaultObjectFieldTemplate, defaultVernaWidgets } from '../templates';
-import VernaJSONSchemaType from '../types/rjsf';
+import { defaultVernaWidgets } from '../../templates';
+import VernaJSONSchemaType from '../../types/rjsf';
 import {
   cleanUiSchema,
   getSelectedDefaultValues,
   getSelectedSchema,
   getSelectedUiSchema,
-} from '../utils/schema';
-import { TranslationType } from '../types/translations';
+} from '../../utils/schema';
+import { TranslationType } from '../../types/translations';
```

Comment:
```suggestion
import { TranslationType } from '../../types/translations';
```
it should be placed after the component import to be in alphabetical order

Reply:
I guess it's what i did, but what part do you check to order alphabetically? The from part?

Reply:
yes the from part